common/api/ecschema-metadata.api.md

```diff
@@ -425,6 +425,8 @@ export enum ECObjectsStatus {
     // (undocumented)
     MissingSchemaUrl = 35070,
     // (undocumented)
+    NewerSchemaVersion = 35081,
```

Comment:
This could be confused with the version of the schema rather than the version of the EC Spec.

Reply:
Yeah, true !
I'll update it to NewerECSpecVersion